frontend/src/components/Farm.js

```diff
@@ -42,24 +47,49 @@ const Farm = ({ farm }) => {
     micropython_array.push(<Snek size="micropython" length={5} up={i%2==0 ? state.up : !state.up}/>)
```

Comment:
This whole section from the console log including "micropython_array" is duplicated - I think it's come back in the merge :/

Reply:
Sorry, my bad! Will remove after work.